_src-js/lib/playlist.js

```diff
@@ -64,11 +64,74 @@ export default [{
   id: 'elephantsdream',
   sources: [
     { src: '//d2zihajmogu5jn.cloudfront.net/elephantsdream/hls/ed_hd.m3u8', type: 'application/x-mpegurl' },
+  ],
+  duration: 653,
+  textTracks: [
+    {
+      src: '//d2zihajmogu5jn.cloudfront.net/elephantsdream/chapters.en.vtt',
+      kind: 'chapters',
+      srclang: 'en',
+      label: 'English'
+    }
+  ],
+  poster: '//d2zihajmogu5jn.cloudfront.net/elephantsdream/poster.png',
+  thumbnail: [
+    {
+      srcset: '//d2zihajmogu5jn.cloudfront.net/elephantsdream/poster.png',
+      type: 'image/png',
+      media: '(min-width: 400px;)'
+    },
+    {
+      src: '//d2zihajmogu5jn.cloudfront.net/elephantsdream/poster.png'
+    }
+  ]
+}, {
+  name: "Elephant's Dream (mp4 with sidecar captions, audio description and chapters)",
```

Comment:
"Elephant's Dream (mp4 with **separate text track audio description, captions,** and chapters)"